Hymns/Common/CustomSearchTransition.swift

```diff
@@ -0,0 +1,9 @@
+import SwiftUI
+
+extension AnyTransition {
+    static var customSearchTransition: AnyTransition {
+        let insertion = AnyTransition.opacity
+        let removal = AnyTransition.move(edge: .trailing)
+        return .asymmetric(insertion: insertion, removal: removal)
+    }
```

Comment:
is it necessary to have this, rather than just to do it inline? I don't foresee this specific animation having much of a reuse case.